### ISSUE TITLE
Render 3D style that is created using mpbox studio 

### ISSUE THREAD

**Issue by ghadeeraqraa1992** [mapbox/mapbox-maps-android#1874]
## Environment

- Android OS version:11
- Maps SDK Version:com.mapbox.maps:android:10.9.0

## Observed behavior and steps to reproduce

We created a custom 3D style Using Mapbox Studio, below is it
<img width="1443" alt="Screen Shot 2022-11-29 at 12 59 40 PM" src="https://user-images.githubusercontent.com/65596246/204512068-dec6f6e8-5fc7-4fc8-b647-b3573df4e382.png">

After loading it using android Mapbox SDK I got the below result, it is like the buildings have been added to the map 

<img width="577" alt="Screen Shot 2022-11-29 at 1 14 06 PM" src="https://user-images.githubusercontent.com/65596246/204514851-9cfa8c32-d45d-48cc-86a9-9884f2192428.png">

## Code
```
      mapView?.getMapboxMap()?.loadStyleUri(getString(R.string.MAP_CUSTOM_3D_STYLE)) {style->
}
```

**Comment by ghadeeraqraa1992** [comment]
Change the pitch and bearing value to give us a better result
`
val initialCameraOptions = CameraOptions.Builder()
  .center(Point.fromLngLat(-74.0066, 40.7135))
  .pitch(45.0)
  .zoom(15.5)
  .bearing(-17.6)
  .build()
` 
https://docs.mapbox.com/android/maps/guides/camera-and-animation/camera/

**Comment by ZiZasaurus** [comment]
@ghadeeraqraa1992 yes, the pitch/ zoom need to be changed in order to properly visualize 3D styles rendered in Mapbox Studio. I attached a screenshot of one of my styles below. 

![Screenshot_1669728693](https://user-images.githubusercontent.com/44972592/204542940-97df8fc8-2089-484f-a25d-a00b492d9eac.png)


For additional reference, please take a look at our Display 3D Building Extrusions example [here](https://docs.mapbox.com/android/maps/examples/fill-extrusion/).
